package.json

```diff
@@ -1,67 +1,66 @@
 {
-  "name": "inversify-express-utils",
-  "version": "6.4.7",
   "author": "Cody Simms",
-  "description": "Inversify wrapper for the express framework",
-  "license": "MIT",
-  "main": "lib/index.js",
-  "module": "es/index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/inversify/inversify-express-utils.git"
-  },
-  "scripts": {
-    "build": "npm run build:lib && npm run build:es && npm run build:es6",
-    "build:lib": "tsc -p src/tsconfig.json",
-    "build:es": "tsc -p src/tsconfig-es.json",
-    "build:es6": "tsc -p src/tsconfig-es6.json",
-    "clean": "rm -r es es6 lib",
-    "test": "jest",
-    "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch",
-    "pretest": "eslint ./",
-    "eslint:fix": "eslint --fix ./",
-    "publish-please": "publish-please",
-    "prepublish": "npm run build && publish-please guard",
-    "update": "updates --update --minor && (git diff-files --quiet package.json || (rimraf package-lock.json node_modules && npm install))",
-    "postupdate": "git diff-files --quiet package-lock.json || npm test"
-  },
   "bugs": {
     "url": "https://github.com/inversify/inversify-express-utils/issues"
   },
-  "homepage": "https://github.com/inversify/inversify-express-utils#readme",
-  "jsnext:main": "es/index.js",
-  "keywords": [
-    "InversifyJS",
-    "express"
-  ],
   "dependencies": {
-    "express": "4.21.1",
-    "http-status-codes": "2.3.0",
-    "inversify": "^6.0.3"
+    "http-status-codes": "2.3.0"
   },
+  "description": "Some utilities for the development of express applications with Inversify",
   "devDependencies": {
     "@types/async": "3.2.24",
     "@types/cookie-parser": "1.4.7",
-    "@types/express": "4.17.21",
+    "@types/express": "^4.17.21",
     "@types/jest": "29.5.14",
-    "@types/node": "22.8.0",
+    "@types/node": "20.17.1",
     "@types/supertest": "6.0.2",
     "@typescript-eslint/eslint-plugin": "8.11.0",
     "@typescript-eslint/parser": "8.11.0",
     "async": "3.2.6",
     "cookie-parser": "1.4.7",
-    "eslint": "8.57.0",
+    "eslint": "9.12.0",
     "eslint-plugin-import": "2.31.0",
     "jest": "29.7.0",
     "publish-please": "5.5.2",
     "reflect-metadata": "0.2.2",
-    "supertest": "7.0.0",
+    "supertest": "6.3.4",
     "ts-jest": "29.2.5",
     "typescript": "5.6.3",
-    "updates": "16.4.0"
+    "updates": "15.3.1"
   },
   "peerDependencies": {
-    "inversify": "6.0.3"
-  }
+    "express": "^4.21.1",
+    "inversify": "^6.0.3"
```

Comment:
I dont know if you notice but also suggested a changed in the format of the package.json, the package info is all over and should be following the standard npm init package generation format

The suggested package order was:

```json
{
  "name": "inversify-express-utils",
  "version": "6.4.7",
  "author": "Cody Simms",
  "description": "Inversify wrapper for the express framework",
  "license": "MIT",
  "main": "lib/index.js",
  "module": "es/index.js",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/inversify/inversify-express-utils.git"
  },
  "scripts": {
    "build": "npm run build:lib && npm run build:es && npm run build:es6",
    "build:lib": "tsc -p src/tsconfig.json",
    "build:es": "tsc -p src/tsconfig-es.json",
    "build:es6": "tsc -p src/tsconfig-es6.json",
    "clean": "rm -r es es6 lib",
    "test": "jest",
    "test:coverage": "jest --coverage",
    "test:watch": "jest --watch",
    "pretest": "eslint ./",
    "eslint:fix": "eslint --fix ./",
    "publish-please": "publish-please",
    "prepublish": "npm run build && publish-please guard",
    "update": "updates --update --minor && (git diff-files --quiet package.json || (rimraf package-lock.json node_modules && npm install))",
    "postupdate": "git diff-files --quiet package-lock.json || npm test"
  },
  "bugs": {
    "url": "https://github.com/inversify/inversify-express-utils/issues"
  },
  "homepage": "https://github.com/inversify/inversify-express-utils#readme",
  "jsnext:main": "es/index.js",
  "keywords": [
    "InversifyJS",
    "express"
  ],
  "dependencies": {
    "express": "4.21.1",
    "http-status-codes": "2.3.0",
    "inversify": "^6.0.3"
  },
  "devDependencies": {
    "@types/async": "3.2.24",
    "@types/cookie-parser": "1.4.7",
    "@types/express": "4.17.21",
    "@types/jest": "29.5.14",
    "@types/node": "22.8.0",
    "@types/supertest": "6.0.2",
    "@typescript-eslint/eslint-plugin": "8.11.0",
    "@typescript-eslint/parser": "8.11.0",
    "async": "3.2.6",
    "cookie-parser": "1.4.7",
    "eslint": "8.57.0",
    "eslint-plugin-import": "2.31.0",
    "jest": "29.7.0",
    "publish-please": "5.5.2",
    "reflect-metadata": "0.2.2",
    "supertest": "7.0.0",
    "ts-jest": "29.2.5",
    "typescript": "5.6.3",
    "updates": "16.4.0"
  },
  "peerDependencies": {
    "inversify": "6.0.3"
  }
}
```

Reply:
Sure, I'll go for it :+1: